Modules/audioop.c

```diff
@@ -65,6 +65,8 @@ search(int16_t val, const int16_t *table, int size)
         if (val <= *table++)
             return (i);
     }
+    assert(0 <= size);
```

Comment:
Maybe put these assertions at the top of the function to get better coverage. (Currently we always just pass `8`.)

Reply:
Oh, wait: I remember that I put them at the bottom exactly because I wanted to minimize coverage to what the code actually required.  But I'll still move them to the top.